js/ui/windowManager.js

```diff
@@ -1552,6 +1552,12 @@ const WindowManager = new Lang.Class({
         }
         currentSession[0].buttonBuilder.destroy();
         currentSession[0].windowBuilder = null;
+
+        if (currentSession[0].windowApp) {
+            currentSession[0].windowApp.get_meta_window().activate(global.get_current_time());
+            currentSession[0].windowApp.show();
+            currentSession[0].buttonApp.show();
```

Comment:
It looks like this will give the app window focus once the builder window loses focus, which doesn't seem right.